content/cloud/noops-sql-db.md

```diff
@@ -1,29 +1,31 @@
 title = "NoOps SQLite Database"
```

Comment:
Checked and we are using "NoOps SQL" as the general name for the feature. We need to update the CLI too
```suggestion
title = "NoOps SQL Database"
```